src/components/ChatEntry.js

```diff
@@ -1,22 +1,28 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
+
+const ChatEntry = ({ sender, body, timeStamp, liked, onLikeClick }) => {
+  const isLocal = sender === 'Vladimir';
 
-const ChatEntry = (props) => {
   return (
-    <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+    <div className={`chat-entry ${isLocal ? 'local' : 'remote'}`}>
+      <h2 className="entry-name">{sender}</h2>
       <section className="entry-bubble">
-        <p>Replace with body of ChatEntry</p>
-        <p className="entry-time">Replace with TimeStamp component</p>
-        <button className="like">🤍</button>
+        <p>{body}</p>
+        <p className="entry-time"><TimeStamp time={timeStamp}></TimeStamp></p>
```

Comment:
When there are no children, you're allowed to use a self-closing tag.
```suggestion
        <p className="entry-time"><TimeStamp time={timeStamp} /></p>
```